.travis.yml

```diff
@@ -17,15 +17,22 @@ notifications:
   email: false
 
 before_script:
-  - go install github.com/golangci/golangci-lint/cmd/golangci-lint
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin v1.12.5
   - go install github.com/onsi/ginkgo/ginkgo  # installs the ginkgo CLI
   - go install github.com/onsi/gomega/...     # fetches the matcher library
 
-
 script:
   - golangci-lint run       # run a bunch of code checkers/linters in parallel
   - ginkgo -r --randomizeAllSpecs --randomizeSuites --failOnPending --cover --trace --race --progress
-  - go build                # build it
+  - GOOS=linux go build # target the container environment
```

Comment:
I don't _think_ the executable is being used for anything other than the container, but pls check me on that